### ISSUE TITLE
Why is SRD0018 not triggered here?

### ISSUE THREAD

**Issue by ErikEJ** [ErikEJ/SqlServer.Rules#720]
```sql
CREATE TABLE #test (
originalId VARCHAR(50),
currentId VARCHAR(50)
);
UPDATE dst
SET currentId = bodyItems.currentId
FROM dbo.test dst
INNER JOIN #test bodyItems
ON bodyItems.originalId = dst.originalId;
```

@niphlod FYI

**Comment by ErikEJ** [comment]
@niphod Because the rule is not triggered when using temp tables

**Comment by niphlod** [comment]
Hmmm. So it's fine to have UPDATE without WHERE? And It popped up only
because of the tablet variabile???

Il dom 2 ago 2026, 08:40 Erik Ejlskov Jensen ***@***.***> ha
scritto:
